config/locales/links/en.yml

```diff
@@ -2,6 +2,7 @@
 en:
   links:
     back_to_sp: "‹ Back to %{sp}"
+    back_to_sp_alt: "Back to %{sp}"
```

Comment:
Yeah the string started out as `No thanks, take me back to SP` that's why I made it separate, but you're right, I can just add the arrow to the view to avoid this duplication. Will update.